configmaptocrs/main.go

```diff
@@ -42,57 +43,59 @@ func main() {
 	log.Printf("MetalLB generator starting. commit: %s branch: %s goversion: %s",
 		version.CommitHash(), version.Branch(), version.GoString())
 
-	f, err := os.Create(filepath.Join(inputDirPath, outPutFileName))
+	res, err := generate(*source)
```

Comment:
I think I remember we said to add a parameter to output the crds to the std output. If that's not the case anymore, the previous version was enough I think, and we only need to add the new mode of parsing of the yaml.

Likewise, if we want to bring the parameter back, passing `os.Stdout` as the writer will make the trick (I think)

Reply:
done